packages/@aws-cdk/aws-codepipeline/lib/common.ts

```diff
@@ -1,10 +1,10 @@
-import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import { ActionArtifactBounds } from './action';
 
 /**
```

Comment:
I thought we resolved this already? 😃

Do not touch this file - leave it where it is, don't move it to the `@aws-cdk/aws-codepipeline` module, and do not make it public.

Reply:
we had, my bad. it's what i get for picking back up after a week or so. apologies. reverted